### DISCUSSION TITLE
关于复制增强来说（或者说单论翻译功能来说）

### DISCUSSION THREAD

**Discussion by orangesxs** [shuax/MouseInc.Settings#406] [Ideas]
我觉得这个复制增强的功能，选择功能是弹出一个悬浮窗式的显示简略信息而不是打开浏览器窗口

![image](https://user-images.githubusercontent.com/96775878/189913057-1359e362-abdc-4eac-916f-e3e3fb4e48fd.png)

就是类似 比如说浏览器里的翻译插件 这种效果是打开一个悬浮弹层而不是打开一个新的浏览器窗口，这样看完结果之后点击浮层外面就直接关闭浮层了，很方便

![image](https://user-images.githubusercontent.com/96775878/189915270-1cd28e0a-4abe-483d-a918-7ff0b24483a3.png)

我觉得这样会更加灵活一些（当然是对于想要快速翻译预览结果的人来说，搜索的话可能打开新窗口会比较实用）
......（像我这种英语渣来说，有时候是遇到单词读不懂了需要翻译一下，看了翻译结果就直接关闭窗口，我感觉是悬浮窗翻译的话会更加灵活，不需反复打开浏览器关闭浏览器）
个人的想法哇

**Comment by wha4up** [comment]
那这样还要重新写个浏览窗口吧？
不过不知道能不能像OCR复制提取文本那样在右下角以通知形式实现。
